ui/src/hooks/useSearchByKeywordData.js

```diff
@@ -16,12 +16,15 @@ function useSearchByKeywordData(query) {
   }
 
   useEffect(() => {
+    // eslint-disable-next-line no-alert
+    const showAlert = () => window.alert('Error occurred while downloading jobs!');
```

Comment:
move `window.alert` to `commons` and name it `showAlertMessage(message)` to avoid multiple `// eslint-disable-next-line no-alert`